src/Common/Commands.Common/Extensions/CmdletExtensions.cs

```diff
@@ -226,6 +226,10 @@ public static string TryResolvePath(this PSCmdlet psCmdlet, string path)
             {
                 return psCmdlet.ResolvePath(path);
             }
+            catch (Exception ex) when (ex is ItemNotFoundException)
+            {
+                return Path.GetFullPath(psCmdlet.SessionState.Path.CurrentLocation + path);
```

Comment:
What is this actually doing?  Can we use Path.Combine instead of string concatenation here?

Reply:
Also, can we add a unit test?